frontend/app/services/lookup-service.server.ts

```diff
@@ -60,9 +73,56 @@ function createLookupService() {
     throw new Error(`Failed to fetch data. Status: ${response.status}, Status Text: ${response.statusText}`);
   }
 
+  /**
+   *
+   * @returns all countries
+   */
+  async function getCountries() {
+    const url = `${INTEROP_API_BASE_URI}/lookups/countries/`;
+    const response = await fetch(url);
+
+    const countryListSchema = z.array(countrySchema);
+    if (response.ok) return countryListSchema.parse(await response.json());
+
+    logger.error('%j', {
+      message: 'Failed to fetch data',
+      status: response.status,
+      statusText: response.statusText,
+      url: url,
+      responseBody: await response.text(),
+    });
+
+    throw new Error(`Failed to fetch data. Status: ${response.status}, Status Text: ${response.statusText}`);
+  }
+
+  /**
+   *
+   * @returns returns province list
+   */
+  async function getRegions() {
+    const url = `${INTEROP_API_BASE_URI}/lookups/regions`;
+    const response = await fetch(url);
+
+    const regionListSchema = z.array(regionSchema);
+    if (response.ok) return regionListSchema.parse(await response.json());
+    if (response.status === 404) return null;
```

Comment:
This line (`if (response.status === 404) return null;`) can be removed.

The API should return an empty collection if no regions are found. If there the response results in a 404, we will throw instead.

Reply:
removed